commit 6177c2188b9c2a799cd92d7c2ffec50a50cbe13a (refs/changes/12/2012/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-05-29T10:58:02-07:00 (2 years, 4 months ago)
    
    IMGAPI-627 switch standalone IMGAPI to using ECDSA keys (latest smtools for DATASET-1269 fix for standalone IMGAPI)

diff --git a/targets.json.in b/targets.json.in
index 66c6015..6df3b2f 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -651,6 +651,7 @@ cat <<EOF
         "dateutils-0.3.1nb1",
         "haproxy-1.6.2",
         "openssl-1.0.2j",
+        "smtools-20160926",
         "stud-0.3p53nb5",
         "the_silver_searcher-0.31.0",
         "xz-5.2.2"
